src/Cli/dotnet/commands/dotnet-workload/install/FileBasedInstaller.cs

```diff
@@ -348,6 +357,26 @@ public void GarbageCollectInstalledWorkloadPacks(DirectoryPath? offlineCache = n
                     Directory.Delete(packIdDir);
                 }
             }
+
+            if (cleanAllPacks)
+            {
+                EradicateAllWorkloadInstallationRecords(_sdkFeatureBand);
+            }
+        }
+
+        public void EradicateAllWorkloadInstallationRecords(SdkFeatureBand currentFeatureBand)
```

Comment:
Looks like the parameter isn't used any more so we can just delete it, right?

Also, I'd suggest using "Delete" or "Remove" in the name instead of "Eradicate".

```suggestion
        public void RemoveAllWorkloadInstallationRecords()
```

Reply:
Thank you! Yeah, that function name was not great.